ramalama

```diff
@@ -333,6 +333,48 @@ def pull_cli(ramalama_store, args, port):
 funcDict["pull"] = pull_cli
 
 
+def push_oci(model, ramalama_store):
+    # Remove the prefix and extract target details
+    target = re.sub(r'^oci://', '', model)
+    registry, reference = target.split('/', 1)
+    registry, reference = ("docker.io", target) if "." not in registry else (
+        registry, reference)
+    reference_dir = reference.replace(":", "/")
+
+    # Validate the model exists locally
+    local_model_path = os.path.join(
```

Comment:
To do: we should be able to push any model to oci registries, including huggingface and ollama